ee/modules/110-istio/openapi/config-values.yaml

```
@@ -138,6 +138,14 @@ properties:
         description: Turn on or off tracing collection and displaying in kiali.
         default: false
         x-examples: [true]
+      sampling:
+        type: number
+        minimum: 0.01
+        maximum: 100.0
+        multipleOf: 0.01
+        description: Value of sampling rate in percents. The sampling rate should be in the range of 0.0 to 100.0 with a precision of 0.01.
```

```suggestion
        description: |
          Sampling rate in percents. The rate should be in the range of 0.0 to 100.0 with a precision of 0.01. It is possible to override this setting with Pod annotation:
          ```
          proxy.istio.io/config: |
            tracing:
              sampling: 100.0
          ```
```